stdlib/cmake/modules/AddSwiftStdlib.cmake

```diff
@@ -2373,8 +2376,10 @@ function(add_swift_target_library name)
         if (SWIFTLIB_BACK_DEPLOYMENT_LIBRARY)
           # Back-deployment libraries get installed into a versioned directory.
           set(install_dest "lib${LLVM_LIBDIR_SUFFIX}/${resource_dir}-${SWIFTLIB_BACK_DEPLOYMENT_LIBRARY}/${resource_dir_sdk_subdir}")
-        else()
+        elseif(sdk STREQUAL WINDOWS OR sdk IN_LIST SWIFT_DARWIN_PLATFORMS)
```

Comment:
I generally kept things the same on Windows, even though it already moved to architecture-specific directories. @compnerd, if you want me to consolidate Windows more, as I did in the compiler, let me know.

Reply:
I am a fan of having fewer cases, as long as things continue to work, I don't mind seeing things becoming uniform :)

Reply:
I believe Windows is no different from Generic Unix here, so let’s make Darwin the only special case.

Reply:
I'm not sure what Windows needs, which is why I mentioned this. Before, all three of Windows, Darwin, and Unix were installing here to `toolchain-os-arch/usr/lib/swift/os`, which is only done when cross-compiling the stdlib for Windows because the first block above is invoked instead when natively compiling the stdlib on Windows.

Now that I'm moving Unix to this architecture-specific install directory, I wasn't sure if the same would work when cross-compiling the Windows stdlib, so I left it the same. If you think that will work for cross-compiling the Windows stdlib too, I will change it, but it will be untested, unless one of you tries it.

Reply:
I think that we should assume that it will be the same and should be safe to make this `elseif (sdk IN_LIST SWIFT_DARWIN_PLATFORMS)`.

Reply:
OK, can you add that later for Windows? I don't want to kick off another whole CI run for this unimportant change, would rather just get these pulls in first.